specifyweb/frontend/js_src/lib/components/Header/NotificationRenderers.tsx

```diff
@@ -23,7 +23,7 @@ export const notificationRenderers: IR<
       <>
         {notificationsText.feedItemUpdated()}
         <Link.Green
-          className="w-fit"
+          className="w-fit normal-case"
```

Comment:
This looks good. Are there any other links/buttons in that file that should also have `normal-case` added to them?